ngi_pipeline/engines/piper_ngi/local_process_tracking.py

```diff
@@ -441,12 +440,16 @@ def record_process_sample(project, sample, workflow_subtask, analysis_module_nam
         if workflow_subtask == "merge_process_variantcall":
             sample_status_field = "analysis_status"
             sample_status_value = "UNDER_ANALYSIS"
+            sample_data_status_field = "status"
+            sample_data_status_value = "STALE"
             seqrun_status_field = "alignment_status"
             seqrun_status_value = "RUNNING"
             extra_args = {"mean_autosomal_coverage": 0}
         elif workflow_subtask == "genotype_concordance":
             sample_status_field = seqrun_status_field = "genotype_status"
             sample_status_value = seqrun_status_value = "UNDER_ANALYSIS"
+            sample_data_status_field = "status"
+            sample_data_status_value = "STALE"
```

Comment:
I guess it is not run at all at the moment, but when it is, I would say that the STALE/FRESH status should not be changed because of it. @vezzi or @parlundin could perhaps confirm this